docs/integrations/dbt.md

```diff
@@ -43,7 +43,9 @@ Consider the following when using a dbt project:
 
 ## How to use SQLMesh incremental models with dbt projects
 
-Incremental loading is a powerful technique when datasets are large and recomputing tables is expensive. SQLMesh offers first-class support for incremental models, and its approach differs from dbt's in that SQLMesh will always run incrementally, even the first time, detecting and backfilling each missing time interval.
+Incremental loading is a powerful technique when datasets are large and recomputing tables is expensive. SQLMesh offers first-class support for incremental models, and its approach differs from dbt's. 
+
+dbt's actions differ based on whether an incremental model has been run before. The first time a model runs dbt computes the transformations for all time intervals at once, which can be problematic if the table is large. In subsequent model runs dbt transforms only the new time intervals. These differences in behavior mean you must know whether a model has already run to predict what will happen in a new run. SQLMesh incremental models allow you to specify how incremental computations are executed with the [`batch_size`](../concepts/models/overview#batch_size) parameter, backfilling missing time intervals in increments of your choice.
```

Comment:
I feel like this section is going to confuse more than help. What if we said something like: "This section describes how to adapt dbt's incremental models to run on sqlmesh and still maintain backwards compatibility with dbt."